src/libraries/System.Net.Security/src/System/Net/Security/SslConnectionInfo.Unix.tt

```diff
@@ -0,0 +1,88 @@
+﻿<#@ template debug="false" hostspecific="false" language="C#" #>
+<#@ import namespace="System" #>
+<#@ assembly name="System.Core" #>
+<#@ assembly name="System.Net.Primitives" #>
+<#@ import namespace="System.Linq" #>
+<#@ import namespace="System.Text" #>
+<#@ import namespace="System.Collections.Generic" #>
+<#@ output extension=".cs" #>
+<#@ include file="TlsCipherSuiteNameParser.ttinclude" #><#@
+    include file="TlsCipherSuite.cs" #>
+<# Array tlsEnumValues = typeof(TlsCipherSuite).GetEnumValues(); #>
+<# Array exchangeEnumValues = typeof(ExchangeAlgorithmTypeIndex).GetEnumValues(); #>
+<# Array cipherEnumValues = typeof(CipherAlgorithmTypeIndex).GetEnumValues(); #>
+<# Array hashEnumValues = typeof(HashAlgorithmTypeIndex).GetEnumValues(); #>
+
+
+using System.Diagnostics;
+using System.Security.Authentication;
+
+namespace System.Net.Security
+{
+    internal partial struct SslConnectionInfo
+    {
+        private void MapCipherSuite(TlsCipherSuite cipherSuite)
+        {
+            TlsCipherSuite = cipherSuite;
+            KeyExchKeySize = 0;
```

Comment:
```suggestion
```

Reply:
@jkotas, would it work with the trimmed apps? I read somewhere that explicit initialization of fields is necessary with trimmed apps. (`struct` fields maybe exempted?)

Reply:
> I read somewhere that explicit initialization of fields is necessary with trimmed apps.

I do not think there is any rule like that. Trimming does not change what needs or does not need to be explicitly initialized.

Looking at this again, I think it is fine to keep the explicit initialization to 0 for clarity here. You can ignore this suggestion.

Reply:
> I do not think there is any rule like that. Trimming does not change what needs or does not need to be explicitly initialized.

I was talking about https://github.com/dotnet/runtime/pull/37541.

Reply:
> > I do not think there is any rule like that. Trimming does not change what needs or does not need to be explicitly initialized.
> 
> I was talking about #37541.

That PR was switching the repo to use the officially supported SkipLocalsInitAttribute (that was new at that time) instead of a previous hack that was achieving the same thing.

We'd previously have an undocumented switch in ILLinker to flip localsinit bit on method bodies in the rewritten assemblies. It's not related to trimming besides the fact that this was slammed into ILLinker because it's an IL rewriter that we were already running as part of the repo build and it was cheaper than writing a new tool just to flip the localsinit bits.

ILLinker deleted support for this switch in the meantime as well.